modules/500-upmeter/monitoring/prometheus-rules/alerting.yaml

```diff
@@ -215,3 +215,258 @@
         summary: Smoke-mini is unavailable
         description: |
           Smoke-mini is unavailable. Please check the linked alerts to find the real problem.
+
+- name: d8.upmeter.resources
+  rules:
+    - alert: D8UpmeterProbeGarbageConfigmap
+      expr: |
+        (
+          min_over_time  (count (kube_configmap_info{namespace="d8-upmeter", configmap=~"upmeter-basic-.*"})[5m:])
+          /
+          avg_over_time  (count (kube_pod_labels{namespace="d8-upmeter", label_app="upmeter-agent"})[5m:])
+        ) >= 1
```

Comment:
I would simplify the rule to make it less complicated and avoid false positives. Let's say 15 configmaps in the namespaces is a problem.  Just sharing my thoughts, you do not need to change the current logic.